language/sygus.tex

```diff
@@ -2788,6 +2919,7 @@ \section{Reserved Words}%
 $\forallkwd$,
 $\constraintinvkwd$,
 $\letkwd$,
+$\optsynthkwd$,
```

Comment:
[minor]

Please also add `optimize-synth` to `keywords=[1]` in "listing-style.tex" for syntax highlighting.

Reply:
I noticed that you have added a new `keywords=[10]` set. I'm sorry about the confusion here. I recently pushed a minor commit with a bunch of changes, and one of them was renumbering the keywords, and now `keywords=[10]` is already defined. You would notice this on rebasing your branch against current `master`.